src/Fuzz.elm

```diff
@@ -241,6 +241,14 @@ string =
         Shrink.string
 
 
+{-| A fuzzer for string values. Generates random printable ascii strings whose
+length can be specified.
+-}
+longString : Int -> Fuzzer String
+longString maxLen=
```

Comment:
maybe `stringOfLength : Int -> Int -> Fuzzer String`

which will then be used like this:

`fuzzString = stringOfLength 2 12`